### PR TITLE
Fix the bug that configs of V4DBHandler on DB are not deleted when dropping tables

### PR DESCRIPTION
**Pull request by d-hayashi**
## What?
This patch includes the following bug things:
- Add function `exist_table` to check the existence of tables on DB
- Support deleting tables which are used for storing configs of V4DBHandler
- Add feature to load configs when `V4DBHandler.read()` is called
- Deprecate the feature of specifying df_name in `V4DBHandler.read()` as it may cause unexpected behavior
- Fix `test_db.py` to clean the environment on each test

## Why?
To fix dataware-tools/dataware-tools#119

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/108#" title="Author self-approved">d-hayashi</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yadach — APPROVED** [review]
lgtm

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/108#" title="Author self-approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/pydtk/pull/108#pullrequestreview-859530259" title="Approved">yadach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi,yadach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
